### DISCUSSION TITLE
How to get bindings value in query.

### DISCUSSION THREAD

**Discussion by joeytzy** [Budibase/budibase#11695] [Help] [answered]
So i created a postgres query in budibase and was wondering how to assign the bindings inside the query.

![image](https://github.com/Budibase/budibase/assets/44018390/c317ff2a-bd79-48d3-b3b8-9f20bd420232)

Im trying to use where clause in the query using the bindings value which are set in the screen when the action is being triggered

**Comment by renrom** [comment, marked as answer]
instead of no_identiti=ic; you identify the binding name as: no_identitit={{ ic }}

**Reply by joeytzy** [reply]
thanks. It worked. I cant find the instructions on this so maybe can make the instructions clearer

**Reply by renrom** [reply]
Hi Joeytzy, i am not a maintainer, just a user as you :). But i think in about 5 clicks there is some clear documentation about bindings:

https://docs.budibase.com/docs/form-bindings#create-query-mapping-query-bindings-to-form-fields

https://docs.budibase.com/docs/introduction-to-bindings

But good to see it works!

**Reply by cl-provecto** [reply]
Would you help me? Using this same example, how would I use a field for Date

**Reply by renrom** [reply]
Hi cl-provecto, maybe show some screens or examples what you already have?  I am not activly using budibase anymore, so maybe some of the maintainers could reply :)

**Reply by cl-provecto** [reply]
Thanks for the answer.

After a lot of trying, I managed to find the solution.
As Bindings returned the value as a String, I used a command to transform it into a Date.

He follows:
`EXTRACT(MONTH FROM TO_DATE({{ dataFiltro }}, 'YYYY-MM-DD'))`
